examples/multi-agent-debate/models.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+from typing import Dict
+
+
+class Message(BaseModel):
+    """Represents a single message with a topic and content.
+
+    Attributes:
+        topic (str): The topic of the message.
+        message (str): The content of the message.
+    """
+    topic: str
+    message: str
+
+
+class SystemMessages(BaseModel):
+    """Represents a collection of system messages.
```

Comment:
I suggest also including instructions to limit the responses of both sides to max 3 points expressed as single sentences (or something reasonable) so it's easy for people to try it out without having to go through a lot of slop. You can add instrucs in the readme saying this is configurable and people can increase it to see a realistic debate.